met-web/src/components/common/index.tsx

```diff
@@ -43,6 +43,24 @@ export const StyledSecondaryButton = styled(MuiButton)(() => ({
     },
 }));
 
+export const StyledGreyButton = styled(MuiButton)(() => ({
+    backgroundColor: 'transparent',
+    color: Palette.text.primary,
+    border: `2px solid ${Palette.text.primary}`,
+    '&:hover': {
+        opacity: '0.8',
+        textDecoration: 'underline',
+        backgroundColor: Palette.primary.main,
+        color: '#FFFFFF',
+    },
+}));
+
+export const GreyButton = ({ children, ...rest }: { children: React.ReactNode; [prop: string]: unknown }) => (
```

Comment:
not sure about this. Buttons in MET should follow government standards, so the existing Primary and Secondary buttons should be used.  We can check with MA

Reply:
well revert it for now no problem